modules/ossm-federation-create-meshPeer.adoc

```diff
@@ -0,0 +1,86 @@
+////
+
+[id="ossm-federation-create-peer_{context}"]
+= Creating a ServiceMeshPeer resource
+
+.Prerequisites
+
+* Two or more {product-title} 4.6 or above clusters.
+* The clusters must already be networked.
+* The load balancers supporting the services associated with the federation gateways must be configured to support raw TLS traffic.
+* Each cluster must have a version 2.1 `ServiceMeshControlPlane` configured to support federation deployed.
+* An account with the `cluster-admin` role.
+
+////
+.Procedure from the Console
+This is conjecture about what the flow might look like…
+
+Follow this procedure to create a `ServiceMeshPeer` resource from the console. This example shows the `red-mesh`  creating a peer resource for the `green-mesh`.
+
+. Log in to the {product-title} web console as a user with the cluster-admin role.
+. Navigate to *Operators* → *Installed Operators*.
+. Click the *Project* menu and select the project where you installed the control plane for the mesh that is creating the `ServiceMeshPeer` resource. For example, `red-mesh-system`.
+. Click the {ProductName} Operator, then click *Istio Service Mesh ServiceMeshPeer*.
+. On the *Istio Service Mesh ServiceMeshPeer* tab, click *Create ServiceMeshPeer*.
+. On the *Create ServiceMeshPeer* page, click *YAML* to modify your configuration.
+. Modify the default configuration with values for the mesh federation between the peers.
+. Click *Create*. The Operator creates the mesh peer based on your configuration parameters.
+. To verify the `ServiceMeshPeer` resource was created, click the *Istio Service Mesh ServiceMeshPeer* tab.
+.. Click the name of the new `ServiceMeshPeer`, for example, `green-mesh`.
+.. Click the *Resources* tab to see the `ServiceMeshPeer` resource the Operator created and configured.
+////
+
+.Procedure from the CLI
+
+Follow this procedure to create a `ServiceMeshPeer` resource from the command line. This example shows the `red-mesh`  creating a peer resource for the `green-mesh`.
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.
++
+[source,terminal]
+----
+$ oc login --username=<NAMEOFUSER> <API token> https://{HOSTNAME}:6443
+----
++
+. Change to the project where you installed the control plane, for example, `red-mesh-system`.
++
+[source,terminal]
+----
+$ oc project red-mesh-system
+----
++
+. Create a `ServiceMeshPeer` file based the following example for the two meshes that you want to federate.
++
+.Example ServiceMeshPeer resource for red-mesh to green-mesh
+[source,yaml]
+----
+kind: ServiceMeshPeer
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: green-mesh
+  namespace: red-mesh-system
+spec:
+  remote:
+    addresses:
+    - ingress-red-mesh.red-mesh-system.apps.domain.com
+  gateways:
+    ingress:
+      name: ingress-green-mesh
+    egress:
+      name: egress-green-mesh
+  security:
+    trustDomain: green-mesh.local
```

Comment:
@JStickler line 74 is not correct. It should be line 74     trust and then next line      domain: green-mesh.local.
You can compare that spec with your previous ones above. The correct path should be spec.security.trust.domain